app/knowledge_panels.py

```diff
@@ -1,167 +1,191 @@
+        if self.country is not None:
+            country_code = country_to_ISO_code(value=self.country)
+            url = f"https://{country_code}-en.openfoodfacts.org"
+            description += f"{self.country} "
+        if self.country is None:
+            url = "https://world.openfoodfacts.org"
+        if self.facet is not None:
+            description += f"{self.facet}"
+        if self.value is not None:
+            query[f"{facet_plural(facet=self.facet)}_tags_en"] = self.value
+            description += f" {self.value}"
+            source_url = f"{url}/{self.facet}/{self.value}?sort_by=last_modified_t"
+        if self.sec_value and self.sec_facet is not None:
+            query[f"{facet_plural(facet=self.sec_facet)}_tags_en"] = self.sec_value
+            description += f" {self.sec_facet} {self.sec_value}"
+            source_url = f"{url}/{self.facet}/{self.value}/{self.sec_facet}/{self.sec_value}?sort_by=last_modified_t"  # noqa: E501
+        expected_html, t_description, t_title = last_edit(url=url, query=query)
 
-def last_edits_kp(
-    facet: str,
-    value: Union[str, None] = None,
-    country: Union[str, None] = None,
-):
-    """
-    Return knowledge panel for last-edits corresponding to different facet
-    """
-    query = {
-        "fields": "product_name,code,last_editor,last_edit_dates_tags",
-        "sort_by": "last_modified_t",
-    }
-    description = ""
-    if facet == "country":
-        country = value
-        country_code = country_to_ISO_code(value=value)
-        url = f"https://{country_code}-en.openfoodfacts.org"
-        facet = value = None
-    if country is not None:
-        country_code = country_to_ISO_code(value=country)
-        url = f"https://{country_code}-en.openfoodfacts.org"
-        description += f"{country} "
-    if country is None:
-        url = "https://world.openfoodfacts.org"
-    if facet is not None:
-        description += f"{facet}"
-    if value is not None:
-        query[f"{facet_plural(facet=facet)}_tags_en"] = value
-        description += f" {value}"
-    expected_html, t_description, t_title = last_edit(url=url, query=query)
+        return {
+            "LastEdits": {
+                "title": t_title,
+                "subtitle": f"{t_description} {description}",
+                "source_url": source_url,
+                "elements": [
+                    {
+                        "element_type": "text",
+                        "text_element": expected_html,
+                    },
+                ],
+            },
+        }
 
-    return {
-        "LastEdits": {
-            "title": t_title,
-            "subtitle": f"{t_description} {description}",
-            "source_url": f"{url}/{facet}/{value}?sort_by=last_modified_t",
-            "elements": [
-                {
-                    "element_type": "text",
-                    "text_element": expected_html,
-                },
-            ],
-        },
-    }
+    def wikidata_kp(self):
```

Comment:
you have to make wikidata for both facet/value.

A good way to implement that is to have a private method _wikidata_kp(self, facet, value)
and to call it twice (once with main facet, once with second facet) in wikidata_kp, and compose results.